spyder_kernels/utils/nsview.py

```diff
@@ -589,7 +589,8 @@ def is_callable_or_module(value):
 def globalsfilter(input_dict, check_all=False, filters=None,
                   exclude_private=None, exclude_capitalized=None,
                   exclude_uppercase=None, exclude_unsupported=None,
-                  excluded_names=None, exclude_callables_and_modules=None):
+                  excluded_names=None, exclude_callables_and_modules=None,
+                  filter_on=False):
```

Comment:
Lets make this new kwarg follow the default value:

```suggestion
                  filter_on=True):
```